autoreject/tests/test_ransac.py

```diff
@@ -32,20 +32,29 @@ def test_ransac():
     # normal case
     picks = mne.pick_types(epochs.info, meg='mag', eeg=False, stim=False,
                            eog=False, exclude=[])
-
     ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
     epochs_clean = ransac.fit_transform(epochs)
     assert len(epochs_clean) == len(epochs)
+
+    # Pass string instead of array of idx
+    picks = 'eeg'
+    ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
```

Comment:
```suggestion
    ransac = Ransac(picks=picks)
```

better to leave out `random_state` since it shouldn't affect picks selection